subxt/Cargo.toml

```diff
@@ -42,6 +42,9 @@ web = [
     "instant/wasm-bindgen"
 ]
 
+# Enable this to use the reconnecting rpc client
+reconnecting-rpc-client = ["dep:reconnecting-jsonrpsee-ws-client"]
```

Comment:
Should we prefix with "unstable" for now until we've settled on the semantics of it (ie probably that the client will reconnect butnot auto resubscribe etc) and handled the DisconnectedWillReconnect?